File: Web/appserver/src/main/java/Utils/Queue/AppServerRequest.java
package Utils.Queue;

public class AppServerRequest
{
	public String requestBody;
	public int pageIndex;
	public int pageSize;
		
	public String responseQueue;
	public int responseTTL;
	
	public AppServerRequest(String requestBody, int pageIndex, int pageSize,
			String responseQueue, int responseTTL) {
		super();
		this.requestBody = requestBody;
		this.pageIndex = pageIndex;
		this.pageSize = pageSize;
		this.responseQueue = responseQueue;
		this.responseTTL = responseTTL;
	}
}

File: Device/model/appserver_request.py
import sys
import json

class AppServerRequest():
    responseQueue = ''
    requestBody = ''
    pageIndex = 0
    pageSize = 0

    def __init__(self,responseQueue = '',requestBody = '',pageIndex = 0,pageSize = 0):
        self.responseQueue = responseQueue
        self.requestBody = requestBody
        self.pageIndex = pageIndex
        self.pageSize = pageSize
       
class AppServerRequestFactory():
    @staticmethod
    def fromJson(jsonString):
        return AppServerRequest(**json.loads(jsonString))

